cypress/integration/common/steps.js

```diff
@@ -363,95 +363,106 @@ Then("there are no notifications in the top menu", () => {
   cy.get(".notifications-menu").should("contain", "0");
 });
 
-Given("there is an annoying user called {string}", (name) => {
+Given("there is an annoying user called {string}", name => {
   const annoyingParams = {
-    email: 'spammy-spammer@example.org',
-    password: '1234',
-  }
-  cy.factory().create('User', {
+    email: "spammy-spammer@example.org",
+    password: "1234"
+  };
+  cy.factory().create("User", {
     ...annoyingParams,
-    id: 'annoying-user',
+    id: "annoying-user",
     name
-  })
-})
-
-Given("I am on the profile page of the annoying user", (name) => {
-  cy.openPage('/profile/annoying-user/spammy-spammer');
-})
-
-When("I visit the profile page of the annoying user", (name) => {
-  cy.openPage('/profile/annoying-user');
-})
-
-When("I ", (name) => {
-  cy.openPage('/profile/annoying-user');
-})
-
-When("I click on {string} from the content menu in the user info box", (button) => {
-  cy.get('.user-content-menu .content-menu-trigger')
-    .click()
-  cy.get('.popover .ds-menu-item-link')
-    .contains(button)
-    .click()
-})
-
-When ("I navigate to my {string} settings page", (settingsPage) => {
+  });
+});
+
+Given("I am on the profile page of the annoying user", name => {
+  cy.openPage("/profile/annoying-user/spammy-spammer");
+});
+
+When("I visit the profile page of the annoying user", name => {
+  cy.openPage("/profile/annoying-user");
+});
+
+When("I ", name => {
+  cy.openPage("/profile/annoying-user");
+});
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-08-19T14:22:58Z" title="Monday, August 19th 2019, 4:22:58 pm +02:00">Aug 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -363,95 +363,106 @@ Then("there are no notifications in the top menu", () => {
   cy.get(".notifications-menu").should("contain", "0");
 });
 
-Given("there is an annoying user called {string}", (name) => {
+Given("there is an annoying user called {string}", name => {
   const annoyingParams = {
-    email: 'spammy-spammer@example.org',
-    password: '1234',
-  }
-  cy.factory().create('User', {
+    email: "spammy-spammer@example.org",
+    password: "1234"
+  };
+  cy.factory().create("User", {
     ...annoyingParams,
-    id: 'annoying-user',
+    id: "annoying-user",
     name
-  })
-})
-
-Given("I am on the profile page of the annoying user", (name) => {
-  cy.openPage('/profile/annoying-user/spammy-spammer');
-})
-
-When("I visit the profile page of the annoying user", (name) => {
-  cy.openPage('/profile/annoying-user');
-})
-
-When("I ", (name) => {
-  cy.openPage('/profile/annoying-user');
-})
-
-When("I click on {string} from the content menu in the user info box", (button) => {
-  cy.get('.user-content-menu .content-menu-trigger')
-    .click()
-  cy.get('.popover .ds-menu-item-link')
-    .contains(button)
-    .click()
-})
-
-When ("I navigate to my {string} settings page", (settingsPage) => {
+  });
+});
+
+Given("I am on the profile page of the annoying user", name => {
+  cy.openPage("/profile/annoying-user/spammy-spammer");
+});
+
+When("I visit the profile page of the annoying user", name => {
+  cy.openPage("/profile/annoying-user");
+});
+
+When("I ", name => {
+  cy.openPage("/profile/annoying-user");
+});
```


Not sure what you are doing here... it's the same as the one above, right?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-20T05:26:45Z" title="Tuesday, August 20th 2019, 7:26:45 am +02:00">Aug 20, 2019</time>_

---


We didn’t do and haven't seen this, but you are absolut right, in my eyes … @alina-beck

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-20T06:11:26Z" title="Tuesday, August 20th 2019, 8:11:26 am +02:00">Aug 20, 2019</time>_

---


sorry @alina-beck these are just linting changes, and make it confusing for the `PR` reviewer

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-08-20T09:41:10Z" title="Tuesday, August 20th 2019, 11:41:10 am +02:00">Aug 20, 2019</time>_

---


Oh, you're right – sorry... I didn't check the file name and was already wondering what this PR had to do with annoying users :wink: